link/perf_event.go

```diff
@@ -432,3 +432,17 @@ func isValidTraceID(s string) bool {
 
 	return true
 }
+
+// getTracefsPath will return a correct path to the tracefs mount point.
+// Since kernel 4.1 tracefs should be mounted by default at /sys/kernel/tracing,
+// but may be also be available at /sys/kernel/debug/tracing if debugfs is mounted.
+// The available tracefs paths will depends on distribution choices.
+func getTracefsPath() string {
+	if tracefsPath == "" {
+		if _, err := os.Stat("/sys/kernel/tracing/kprobe_events"); err == nil {
```

Comment:
Why not stat just `/sys/kernel/tracing`?

Reply:
Sadly on some distro (like Debian 10 and Ubuntu 18 at least) tracefs is only mounted once in /sys/kernel/debug/tracing, but the mount point /sys/kernel/tracing still exist (but empty..). It really depends on each distributions/version init (it's well mounted twice in Ubuntu 20/22).
That's why I stat the kprobe_events entry instead of just the mount point